backend/src/main/java/de/nordakademie/iaa/noodle/dao/ResponseRepository.java

```diff
@@ -12,7 +14,16 @@
 @RepositoryDefinition(idClass = Long.class, domainClass = Response.class)
 @Transactional(propagation = Propagation.REQUIRED)
 public interface ResponseRepository {
-    Response findById(Long id);
+
+    @Query("SELECT response FROM Response response WHERE response.id = ?1 AND response.participation.survey.id = ?2")
```

Comment:
Is the manual JPQL even necessary here?

Reply:
I tried to do it with the repository function name but I did not find a way to do that. If you know how that can be done, we should use that.